Generals/Code/GameEngine/Source/GameClient/MessageStream/LookAtXlat.cpp

```diff
@@ -360,10 +361,16 @@ GameMessageDisposition LookAtTranslator::translateGameMessage(const GameMessage
 			if (m_isRotating)
 			{
 				const Real FACTOR = 0.01f;
+				Real angle = FACTOR * (m_currentPos.x - m_originalAnchor.x);
+				Real targetAngle = m_anchorAngle + angle;
 
-				Real angle = FACTOR * (m_currentPos.x - m_anchor.x);
+				if (TheInGameUI->isInForceAttackMode())
```

Comment:
// TheSuperHackers @tweak Stubbjax 04/08/2025 Snap angle to nearest 45 degrees
				// while using force attack mode for convenience.